docs/docs/30-how-to-guides/40-managing-projects.md

```diff
@@ -4,3 +4,39 @@ sidebar_label: Managing Projects
 ---
 
 # Managing Projects
+
+### Namespace Creation and Adoption in Kargo Projects
+
+When a new Kargo `Project` is created, it automatically generates a corresponding Kubernetes `namespace`.
+In scenarios where specific configuration requirements are needed, Kargo offers an adoption feature for
+pre-existing namespaces.
+
+Kargo can adopt namespaces that you create beforehand, if they are labeled
+with `kargo.akuity.io/project: "true"`.
```

Comment:
This feels a little repetitious. The previous paragraph basically said the same thing, but with less detail. Maybe kill the previous one and keep this one.